schedule/yast/encryption/lvm_encrypt_separate_boot_pvm.yaml

```diff
@@ -27,7 +27,7 @@ schedule:
   - installation/authentication/default_user_simple_pwd
   - installation/installation_overview
   - installation/bootloader_settings/disable_plymouth
-  - installation/bootloader_settings/disable_boot_menu_timeout
+  - installation/disable_grub_timeout
```

Comment:
looks like this change was not needed. right?

Reply:
You are right from the point of view that disable_plymouth is using libyui as well. I guess the problem is that I selected the files to change by loooking who is using disable_boot_menu_timeout but not using setup_libyui. Now looking at the YAML again I see a innocent variable "YUI_REST_API: 1" that seems to trigger libyui without explicitly mentioning in the schedule. Now I clearly understand the very deep love for variables... ;-)

Reply:
could you do a quick revert of this line?

Reply:
Ok, pushed a commit with reverting this to the branch.